test/defaults.test.js

```diff
@@ -76,4 +76,34 @@ describe('defaults', function() {
       });
     });
   });
+
+  context('defaultValueHandling', function() {
+    it('if not set, should not affect default behavior', function(done) {
+      const Apple = db.define('Apple', {
+        color: {type: String, default: 'red'},
+        taste: {type: String, default: 'sweet'},
+      }, {applyDefaultsOnReads: false});
+
+      Apple.create(function(err, apple) {
+        apple.color.should.equal('red');
+        apple.taste.should.equal('sweet');
+        done();
+      });
+    });
+
+    it('if set to "ignore", should remove the property', function(done) {
+      const Apple = db.define('Apple', {
+        color: {type: String, default: 'red', defaultValueHandling: 'ignore'},
+        taste: {type: String, default: 'sweet'},
+      }, {applyDefaultsOnReads: false});
```

Comment:
Should we use the same name `applyDefaultsOnReads` at property level instead of introducing `defaultValueHandling`? For example:
```
const Apple = db.define('Apple', {
        color: {type: String, default: 'red', applyDefaultsOnWrites: false},
        taste: {type: String, default: 'sweet'},
      }, {applyDefaultsOnReads: false});
```

Reply:
`applyDefaultsOnReads` for reading, this feature is related to writing. So maybe `applyDefaultsOnWrite`?